packages/Operators/test/tstPointCloudOperators.cpp

```diff
@@ -104,22 +114,24 @@ TEUCHOS_UNIT_TEST( SplineInterpolationOperator, spline_radius_test )
 	coords[1] = (double) std::rand() / (double) RAND_MAX;
 	coords[2] = (double) std::rand() / (double) RAND_MAX;
 	domain_points[i] = DataTransferKit::Point( point_id, comm_rank, coords );
-	domain_data[i] = coords[0];
+	domain_data[i] = coords[0] + coords[1] + coords[2];
```

Comment:
it was a better test because it checks all three spatial coordinates - not just one